go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/scylladb/scylla-operator
 
-go 1.22.3
+go 1.23
```

Comment:
is removing the patch deliberate?

Reply:
yep, any 1.23 is good enough to build this at this point